docs/tags.md

```diff
@@ -74,3 +74,25 @@ A collision happens when we rename a tag and there is already a tag with the sam
 Steps to merge “dog” tag into "cat" tag:
 * Iterate over the notes replacing "dog" tag with "cat" tag. If the note already has "cat" tag, remove "dog" tag from the note.
 * Delete "dog" tag.
+
+---
+
+## Tag issues
+
+### Tag entity stored with wrong entity id
+
+Tag entities are referenced by their tag hash. The canonical tag name is one of many lexical variations that collapse to the same tag hash. In many accounts we will find inconsistencies in the database because the apps did not always properly store tags. These inconsistencies can lead to issues in the UI, such as showing the wrong subset of notes for a given tag.
+
+Consider the tag `Bücher` whose entity id should be `b%C3%BCcher`. If stored as `Bücher` we might end up with two tag entities whose tag hashses collapse to the same value. This would appear as a tag collision.
+
+In these cases we ought to resolve the inconsistency by removing the tag with the wrong entity id. We need to be carerful when doing this to make sure that the stored tag name in the inconsistent tag entity is an actual lexical variation of the tag's canonical name. It's possible that through renaming a tag we could end up with a wrose issue, which is that the name of the tag collapses to a tag hash for a different tag name.
```

Comment:
```suggestion
In these cases we ought to resolve the inconsistency by removing the tag with the wrong entity id. We need to be careful when doing this to make sure that the stored tag name in the inconsistent tag entity is an actual lexical variation of the tag's canonical name. It's possible that through renaming a tag we could end up with a worse issue, which is that the name of the tag collapses to a tag hash for a different tag name.
```

It looks like there are two typographical errors in this sentence.

Reply:
thanks for the catch! feel free but not obligated to correct typos when you see them.

Reply:
fixed in 887df8e